spec/rails_helper.rb

```diff
@@ -38,6 +38,8 @@
   # instead of true.
   config.use_transactional_fixtures = true
 
+  # config.include Rails.application.routes.url_helpers, type: :request
```

Comment:
Why have you added this comment?

Not against it if there is a rationale.

Reply:
I believe this was part of an original attempt at the routes for the API. Commented out when I realized I didn't need it but it should be removed!

Reply:
Very easy for things like this to slip through, all good.